quarkus/defaults/src/main/resources/application.properties

```diff
@@ -72,10 +72,10 @@ quarkus.otel.sdk.disabled=false
 # quarkus.otel.traces.sampler=parentbased_always_on
 # quarkus.otel.traces.sampler.arg=1.0d
 
-quarkus.test.integration-test-profile=test
+quarkus.test.integration-test-profile=it
 
 polaris.realm-context.type=default
-polaris.realm-context.realms=realm1,realm2,realm3
+polaris.realm-context.realms=POLARIS
```

Comment:
nit: Was old default upper case?

Reply:
Heh no: the default for prod was `default-realm`:

https://github.com/apache/polaris/blob/aa240e9f058eeae3881ebcfda13475188e62ea2b/polaris-server.yml#L79

And the default for tests was `POLARIS`:

https://github.com/apache/polaris/blob/aa240e9f058eeae3881ebcfda13475188e62ea2b/dropwizard/service/src/test/resources/polaris-server-integrationtest.yml#L104

I thought `POLARIS` was more appropriate as a default than `default-realm`, but happy to change that back to `default-realm` if someone thinks otherwise.